nodelet_topic_tools/include/nodelet_topic_tools/nodelet_lazy.h

```diff
@@ -135,7 +155,7 @@ class NodeletLazy: public nodelet::Nodelet
   /** @brief
     * callback function which is called when new subscriber come
     */
-  virtual void connectionCallback(const ros::SingleSubscriberPublisher& pub)
```

Comment:
This argument isn't used. It's not even in the doxygen documentation.

And I had to remove it because otherwise I can use it with an `image_transport::Publisher` callback.